students/hangu/users/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+
+class User(models.Model):
+    user_name    = models.CharField(max_length=60)
+    user_email   = models.CharField(max_length=60, unique=True)
+    password     = models.CharField(max_length=100)
+    phone_number = models.CharField(max_length=13)
```

Comment:
한구님~ 데이터의 이력 관리를 위해서 created_at, updated_at column을 추가해주세요.

추가하시고 migration 다시 한번 부탁드립니다!
```suggestion
class User(models.Model):
    user_name    = models.CharField(max_length=60)
    user_email   = models.CharField(max_length=60, unique=True)
    password     = models.CharField(max_length=100)
    phone_number = models.CharField(max_length=13)
    created_at   = models.DateTimeField(auto_now_add=True)
    updated_at   = models.DateTimeField(auto_now=True)
```

Reply:
네 감사합니다